app.py

```
@@ -983,11 +1005,109 @@ def get_task(name: str):
+        return jsonify({'success': False, 'error': str(e)}), 500
+
+
+@app.route('/api/system_updates/<operation>/start', methods=['POST'])
+@admin_required
+def api_system_updates_start(operation):
+    try:
+        status = system_updates_manager.start_operation(operation)
+        return jsonify({'success': True, 'operation': status})
+    except ValueError as e:
+        return jsonify({'success': False, 'error': str(e)}), 400
+    except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)